clang/tools/clang-linker-wrapper/OffloadWrapper.h

```diff
@@ -10,7 +10,12 @@
 #define LLVM_CLANG_TOOLS_CLANG_LINKER_WRAPPER_OFFLOAD_WRAPPER_H
 
 #include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/SmallVector.h"
```

Comment:
For sake of uniformity, I think it will be a good idea to create a new .h file SYCLOffloadWrapper.h.

Thanks